lib/go-atscfg/regexrevalidatedotconfig.go

```diff
@@ -41,6 +41,12 @@ const DefaultMaxRevalDurationDays = 90
 const JobKeywordPurge = "PURGE"
 const RegexRevalidateMinTTL = time.Hour
 
+type RevalType string
+
+const MISS = RevalType("MISS")
```

Comment:
Nitpick: Go variables, including consts, should be CamelCase. Should be `const Miss` and `const Stale`.
Personally, for enums I prefer `RevalTypeMiss` and `RevalTypeStale`, just for clarity. Especially since they're scoped to the entire `atscfg` package

Reply:
Okay i can do that.  thanks.